packages/frontend/component/src/ui/safe-area/style.css.ts

```diff
@@ -0,0 +1,22 @@
+import { createVar, style } from '@vanilla-extract/css';
+
+export const topOffsetVar = createVar();
+export const bottomOffsetVar = createVar();
+
+export const safeArea = style({
+  selectors: {
+    '&[data-top]': {
+      paddingTop: `calc(${topOffsetVar} + 12px)`,
+    },
+    '&[data-bottom]': {
+      paddingBottom: `calc(${bottomOffsetVar} + 0px)`,
+    },
+    '&[data-standalone][data-top]': {
+      paddingTop: `calc(env(safe-area-inset-top, 12px) + ${topOffsetVar})`,
```

Comment:
do we need to check if `env` feature is available?

Reply:
I think it's stable enough: [https://caniuse.com/?search=env()](https://caniuse.com/?search=env())